README.md

```diff
@@ -22,7 +22,25 @@ Or install it yourself as:
 
 ## Usage
 
-TODO: Write usage instructions here
+You will need to setup the configuration likely via an initializer:
+
+```ruby
+SlackLogger.configure do |c|
```

Comment:
The configuration block is already in the README. You need to rebase against main.

Reply:
I just merged the one from shawn so now updated here.